vips/vip-160/vip-160.ts

```diff
@@ -0,0 +1,122 @@
+import { ProposalType } from "../../src/types";
+import { makeProposal } from "../../src/utils";
+
+const ACM = "0x4788629ABc6cFCA10F9f969efdEAa1cF70c23555";
+
+const NORMAL_TIMELOCK = "0x939bD8d64c0A9583A7Dcea9933f7b21697ab6396";
+const FAST_TRACK_TIMELOCK = "0x555ba73dB1b006F3f2C7dB7126d6e4343aDBce02";
+const CRITICAL_TIMELOCK = "0x213c446ec11e45b15a6E29C1C1b402B8897f606d";
+const PAUSE_GUARDIAN = "0x1C2CAc6ec528c20800B2fe734820D87b581eAA6B";
+const MANAGER = "0x7B1AE5Ea599bC56734624b95589e7E8E64C351c9";
+
+const SHORTFALL = "0x51dcdD4Ee45c0e8c04DE4C54908C2Ada38fd99B5";
+const RISK_FUND = "0xDF829fB0861DA07a0a935486738dFa65ED007927";
+const PROTOCOL_SHARE_RESERVE = "0x99731dDeA6e0d721dfeb434Cb20b200E72709056";
```

Comment:
We need a PR with these addresses (and the testnet ones) in the docs repo

Reply:
https://github.com/VenusProtocol/venus-protocol-documentation/pull/76